pyhafas/profile/db/helper/request.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+import requests
+
+from pyhafas.profile import ProfileInterface
+from pyhafas.profile.base.helper.request import BaseRequestHelper
+from pyhafas.profile.base.mappings.error_codes import BaseErrorCodesMapping
+from pyhafas.types.hafas_response import HafasResponse
+from tenacity import (
+    retry,
+    retry_if_exception_type,
+    wait_exponential,
+    stop_after_attempt,
+)
+
+
+class DBRequestHelper(BaseRequestHelper):
+    tenacity_multiplier = 20
+    tenacity_retry_attempts = 4
+
+    @retry(
```

Comment:
Is there a special reason for a new external library? It seems like this can be done with requests and urllib3: https://stackoverflow.com/questions/15431044/can-i-set-max-retries-for-requests-request

Reply:
I felt like Tenacity's API is much nicer. If you think that not having an additional dependency is preferable, then I can implement using `urllib3`.

Reply:
Do you have an opinion on this @n0emis?